package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "npm-blue-tape",
+  "description": "The type definition for https://github.com/spion/blue-tape",
+  "version": "0.0.1",
+  "author": "blakeembrey",
```

Comment:
Keep it in `devDependencies`. Locking in VSCode is entirely up to you, but I'm good with it being checked in if you want it to be. As long as tests are using NPM scripts which use the local version, that's all I care about.